utils/appliance/__init__.py

```diff
@@ -241,6 +241,20 @@ def _configure_upstream(self, log_callback=None):
         self.restart_evm_service(log_callback=log_callback)
         self.wait_for_web_ui(timeout=1800, log_callback=log_callback)
 
+    @logger_wrap("Configure GCE IPAppliance: {}")
+    def configure_gce(self, log_callback=None):
```

Comment:
Can we put a message for a TODO in here, eventually we will want this moved out as a function/method that is in the provider class, appliance shouldn't know anything special about GCE (eventually)